packages/react/hover-card/src/HoverCard.tsx

```diff
@@ -271,18 +322,33 @@ const HoverCardContentImpl = React.forwardRef<
       onInteractOutside={onInteractOutside}
       onEscapeKeyDown={onEscapeKeyDown}
       onPointerDownOutside={onPointerDownOutside}
-      onFocusOutside={onFocusOutside}
+      onFocusOutside={composeEventHandlers(onFocusOutside, (event) => {
+        event.preventDefault();
+      })}
```

Comment:
Fixes:
I don't think we need this behaviour anyway as we disable focus access in the cards and we're already handling close on trigger `blur`

https://user-images.githubusercontent.com/11708259/193234945-107aa0a1-b398-4755-a0ea-59f84e74bf23.mp4